### ISSUE TITLE
Lock files not unlinked

### ISSUE THREAD

**Issue by poornima-repo** [pjcj/Devel--Cover#263]
When a database is read from the file system, the lock files are not unlinked. 
The change would be in Devel::Cover::IO::Base module.

**Comment by pjcj** [comment]
I agree that this would be ideal, but is it actually causing any problems the way it is at the moment?

If you have a suggestion, or better still a patch, which could implement this without causing problems amongst multiple processes it would be very well received.

**Comment by eserte** [comment]
Maybe related: MYMETA.json is also read using Devel::Cover::DB::IO::JSON, so there's a MYMETA.json.lock file leftover after Devel::Cover operations (so to have a clean git status, one has to remove the lock file manually, or add it to .gitignore).
